std/algorithm/mutation.d

```diff
@@ -541,12 +541,22 @@ $(HTTP sgi.com/tech/stl/copy_backward.html, STL's copy_backward'):
 /**
 Assigns `value` to each element of input _range `range`.
 
+Alternatively, instead of using single `value` to fill the `range`,
+a `fillter` $(REF_ALTTEXT forward _range, isForwardRange, std,_range,primitives)
+can be provided. The length of `filler` and `range` do not need to match, but
+`filler` must not be empty.
+
 Params:
         range = An
                 $(REF_ALTTEXT input _range, isInputRange, std,_range,primitives)
                 that exposes references to its elements and has assignable
                 elements
         value = Assigned to each element of range
+        filler = The
```

Comment:
The was copied from the previous overload, but I think "An" would be better here.